add-on/chrome/bootstrap.js

```diff
@@ -912,12 +912,13 @@ var WindowListener = {
        * Handles events from gBrowser.
        */
       handleEvent: function(event) {
+
         switch (event.type) {
           case "DOMTitleChanged":
             // Get the new title of the shared tab
             this._notifyBrowserSwitch();
             break;
-          case "TabSelect":
+          case "TabSelect": {
```

Comment:
Oh duh. Here's the migration post saying it was added:

http://eslint.org/docs/user-guide/migrating-to-2.0.0#new-rules-in-eslintrecommended